app/src/main/res/xml/settings.xml

```diff
@@ -25,6 +25,14 @@
         android:layout="@layout/bg_settings_section_security">
         <com.seafile.seadroid2.ui.SwitchPreferenceCompat
             android:defaultValue="false"
+            android:key="client_encrypt_switch_key"
```

Comment:
Why would anyone want to set this to "off"? And why is this this "off" by default?